### ISSUE TITLE
Question about training the models together

### ISSUE THREAD

**Issue by chojw** [bryanyzhu/two-stream-pytorch#40]
Hi, I was wondering can the models be trained together with one single loss instead of trained separately?
I have seen other two-stream UCF codes but all of them seem to train separately and then find the loss by adding the output logits or something like that, but do you think its possible to train at once? has this been done?
and if it has, is there any recommended hyperparameters because it seems that they are different for each modality.

Thanks in advance!

**Comment by bryanyzhu** [comment]
There are some methods do early fusion of two streams and use a single loss, such as

Convolutional Two-Stream Network Fusion for Video Action Recognition
https://arxiv.org/abs/1604.06573

**Comment by chojw** [comment]
Thanks for the reference!
